src/Select/InputWrapper.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+import { statusPropType } from '../common-prop-types.js'
+import { ArrowDown, ArrowUp } from '../icons/Arrow.js'
+import { colors, theme } from '../theme.js'
+
+const InputWrapper = ({
+    open,
+    onToggle,
+    children,
+    tabIndex,
+    error,
+    warning,
+    valid,
+    disabled,
+    dense,
+    className,
+    inputRef,
+}) => {
+    const classNames = cx(className, 'root', {
+        error,
+        warning,
+        valid,
+        disabled,
+        dense,
+    })
+
+    return (
+        <React.Fragment>
```

Comment:
No need for a `Fragment` here, you can just move the styles into the outer `div`

Reply:
Ah yeah you're right. No idea why that was there. Fixed in cd4dd59e6e112884902d7d64e801565d9b6a51de